tests/dataset/test_dataset_profile.py

```diff
@@ -0,0 +1,85 @@
+"""Tests for tasks related creating and updating datasets in Tamr"""
+import pandas as pd
+
+import tamr_toolbox as tbox
+from tamr_toolbox import utils
+from tamr_toolbox.dataset import get_profile
+from tamr_toolbox.utils.testing import mock_api
+from tamr_toolbox.data_io import dataframe
+
+from tamr_unify_client import Client
+
+from tests._common import get_toolbox_root_dir
+
+CONFIG = utils.config.from_yaml(
+    get_toolbox_root_dir() / "tests/mocking/resources/toolbox_test.yaml"
+)
+
+# Create some mock data to upsert into a Tamr dataset:
+DATASET_NAME = "mock_data_profile"
+test_data = {"id": ["0", "1"], "first_name": ["John", "Jane"], "last_name": ["Doe", "Doe"]}
+test_data_df = pd.DataFrame(test_data)
+dataframe.validate(test_data_df)
+
+# Note that some test cases are dependent on previous ones
+# So all tests must be set to enforce_online_test = True or all to False
+enforce_online_test = False
```

Comment:
Let's avoid doing this, since it will make adding more tests difficult.  If at all possible, each test should leave the instance in exactly the state it was before the test was run.

Reply:
So, I based my tests off of the other tests in the dataset module... But if you would like me to delete the dataset at the end of each test, I will add that. My only question about doing so is whether it makes sense to have two tests (therefore, create and delete the dataset each time) or to combine them into a single test as I had originally?